### DISCUSSION TITLE
Issues applying for student developer pack

### DISCUSSION THREAD

**Discussion by dylanrattai** [community/community#137189] [GitHub Education]
### Hi everyone,

I want to apply for the student developer pack, but there are a few issues, I would contact support but you arent allowed to for student developer pack issues anymore.

1. My .edu email is @ndus.edu not @ndsu.edu, so I cant apply for my college
2. I cant upload images of any of my proof, my school ID does not have a date on it, the camera will only let me use my front facing camera (so the picture of my proof is so pixelated you cant read it)
3. It says I am away from campus and asks for why, when I am literally sitting on campus right now.

Is there anything I can do or should I just forget trying to apply at this point?

**Comment by sofiaPalavecino** [comment]
I had the exact same problem with the camera.

**Comment by Proinvestor-code** [comment]
Ask you college/university to send you and enrollment letter Github will accept it. The front facing camera this is annoying but it looks like that's they only way to get it done for now.
